coordinator/tests/services/translation/request/agent-unattended.spec.ts

```diff
@@ -12,28 +12,27 @@ jest.mock("../../../../src/services/communication/ods-client", () => ({
   }
 }))
 
-test("user details contain placeholder data", async () => {
+test("user details populated from header user information", async () => {
   const mockLookupFunction = odsClient.lookupOrganization as jest.Mock
   mockLookupFunction.mockReturnValueOnce(Promise.resolve(mockOrganizationResponse))
 
-  const result = await createAuthorForUnattendedAccess("FTX40", logger)
+  const result = await createAuthorForUnattendedAccess("FTX40", undefined, logger)
 
   const agentPerson = result.AgentPerson
-  expect(agentPerson.id._attributes.extension).toEqual("999999999999")
-  expect(agentPerson.code._attributes.code).toEqual("R9999")
-  expect(agentPerson.telecom[0]._attributes.value).toEqual("tel:01234567890")
+  expect(agentPerson.id._attributes.extension).toEqual("555254240100")
```

Comment:
Echoing Jack here - it would be nice to export these test values from headers.ts